handler/src/main/java/io/netty/handler/ssl/OpenSslEngine.java

```diff
@@ -525,7 +526,10 @@ public synchronized SSLEngineResult wrap(
 
                     pendingNetResult = readPendingBytesFromBIO(dst, bytesConsumed, bytesProduced, status);
                     if (pendingNetResult != null) {
-                        return pendingNetResult;
+                        if (pendingNetResult.getStatus() != Status.OK) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Unnecessary use of fully qualified name 'Status.OK' due to existing static import 'javax.net.ssl.SSLEngineResult.Status.OK' [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3AUnnecessaryFullyQualifiedName)